Packs/FeedTAXII/Integrations/FeedTAXII/FeedTAXII.py

```diff
@@ -933,6 +1003,82 @@ def observable_extract_properties(observable):
     return result
 
 
+def indicator_extract_properties(indicator) -> Dict[str, Any]:
+    """Extracts properties from indicator"""
+    result: Dict[str, Any] = {}
+
+    title = next((c for c in indicator if c.name == 'Title'), None)
+    if title is not None:
+        title = title.text
+        result['stix_indicator_name'] = title
+
+    description = next((c for c in indicator if c.name == 'Description'), None)
+    if description is not None:
+        description = description.text
+        result['stix_indicator_description'] = description
+
+    confidence = next((c for c in indicator if c.name == 'Confidence'), None)
+    if confidence is not None:
+        value = next((c for c in confidence if c.name == 'Value'), None)
+        if value is not None:
+            value = value.text
+            result['confidence'] = value
+
+    indicated_ttp = indicator.find_all('Indicated_TTP')
+    if indicated_ttp:
+        result['ttp_ref'] = []
+        for ttp_value in indicated_ttp:
+            ttp = next((c for c in ttp_value if c.name == 'TTP'), None)
+            if ttp is not None:
+                value = ttp.get('idref')
+                result['ttp_ref'].append(value)
+
+    return result
+
+
+def ttp_extract_properties(ttp, behavior):
```

Comment:
Please add a short docstring and add the returned value typing to the method's signature